src/GridFieldOrderableRows.php

```diff
@@ -579,11 +574,12 @@ protected function executeReorder(GridField $grid, $sortedIDs)
                     $sortterm .= "$col $dir, ";
                 }
             } else {
-                $sortterm = $this->extraSortFields.', ';
+                $sortterm = $this->extraSortFields . ', ';
             }
         }
         $list = $grid->getList();
-        $sortterm .= '"'.$this->getSortTable($list).'"."'.$sortField.'"';
+        $sortterm .= '"' . $sortField . '"';
```

Comment:
Changing to v5 stuff the `$sortterm` var no longer requires the `getSortTable` call. That is unless I have misunderstood how this works (entirely possible). The tests were breaking without this change and now work if that helps